cwms-data-api/src/main/java/cwms/cda/data/dao/TimeSeriesDaoImpl.java

```diff
@@ -880,9 +912,13 @@ private void store(Connection connection, String officeId, String tsId, String u
                 qualityArray[i] = value.getQualityCode();
             }
         }
-
         tsDao.store(connection, officeId, tsId, units, timeArray, valueArray, qualityArray, count,
                 storeRule.getRule(), overrideProtection, versionDate, createAsLrts);
+
+        if(versionDate != null) {
+            CWMS_TS_PACKAGE.call_SET_TSID_VERSIONED(getDslContext(connection,officeId).configuration(),
```

Comment:
This call should be above the store, but we don't want to blow up if this throws an error that the tsid couldn't be found as we might be creating it

Reply:
Done